extensions/card4l-eo/README.md

```diff
@@ -0,0 +1,184 @@
+| access                   | STRONGLY RECOMMENDED. Link to data access information.       | 1.16                |
+| sensor-calibration       | Link to the sensor calibration parameters.                   | 1.11                |
+| radiometric-accuracy     | Link describing the assessed absolute radiometric uncertainty of the version of the data or product. | 1.12                |
+| geometric-correction     | Link to the Geometric Correction algorithm details.          | 1.7                 |
+| elevation-model          | Links to the Digital Elevation Models. Preferably links to a STAC Item with additional metadata for the DEMs. | 1.14                |
+| geometric-accuracy       | Link to documentation of estimate of absolute localization error. | 1.8                 |
+| cloud                    | Link to documentation about the cloud detection.             | 2.5                 |
+| cloud-shadow             | Link to documentation about the cloud shadow detection.      | 2.6                 |
+| snow-ice                 | Link to documentation about the snow and ice mask.           | 2.7 (ST) / 2.8 (SR) |
+| land-water               | Link to documentation about the land and water mask (SR only). | 2.7 (SR)            |
+| atmosphere-emissivity    | **REQUIRED.** Link to documentation about corrections for atmosphere and emissivity (ST only). | 3.2 (ST)            |
+| measurement-nomalisation | Link to documentation about measurement normalisation (SR only). | 3.3 (SR)            |
+| atmospheric-scattering   | **REQUIRED.** Link to documentation about the directional atmospheric scattering algorithms (SR only). | 3.4 (SR)            |
+| water-vapor              | **REQUIRED.** Link to documentation about the water vapour corrections (SR only). | 3.5 (SR)            |
+| ozone                    | Link to documentation about the ozone corrections (SR only). | 3.6 (SR)            |
+
+### STAC Item Assets
+
+Whether the metadata are provided in a single record relevant to all pixels, or separately for each pixel, is at the discretion of the data provider. 
+
+Each of the assets can either be exposed individually or grouped together in any form. In the latter case the role names can simply be merged to a set of unique role names. Roles can also be combined for a single file. For example, a cloud mask which is also including cloud shadows can use the roles `cloud` and `cloud-shadow` for a single file. The `file:values` property can the contain which value(s) correspond to clouds and which value(s) correspond to cloud shadows respectively.
+
+The italic role names are proposed to be the asset's key.
+
+| Role Name(s)                           | Additional properties                                        | Description                                                  | Req.                 |
```

Comment:
This is a useful table, but it's not clear if the Additional Properties are required....I think that is the intention and from what I know of CARD4L.

Reply:
Req are given in the table below

Reply:
Required properties are now bold, too.